jwst/jump/jump.py

```diff
@@ -11,7 +11,11 @@
 def run_detect_jumps(input_model, gain_model, readnoise_model,
                      rejection_thresh, three_grp_thresh, four_grp_thresh,
                      max_cores, max_jump_to_flag_neighbors,
-                     min_jump_to_flag_neighbors, flag_4_neighbors):
+                     min_jump_to_flag_neighbors, flag_4_neighbors,
+                     after_jump_flag_dn1=0.0,
```

Comment:
also you don't need to set the defaults here since they are set in 'spec'

Reply:
Like in stcal, the defaults are needed for the tests to run.  Otherwise, these 4 parameters need to be set in all the tests.